instrumentation/google.golang.org/grpc/otelgrpc/interceptor.go

```diff
@@ -347,17 +350,28 @@ func UnaryServerInterceptor(opts ...Option) grpc.UnaryServerInterceptor {
 		)
 		defer span.End()
 
+		meter := cfg.MeterProvider.Meter(
+			instrumentationName,
+			metric.WithInstrumentationVersion(SemVersion()),
+		)
+		rpcServerDuration, _ := meter.SyncFloat64().Histogram("rpc.server.duration")
```

Comment:
Can this be created and live outside the message handling flow?  There should be no need to recreate the meter and instrument on every request.

Reply:
That makes sense. What about the `tracer`? Should it also live outside handling flow? @Aneurysm9

Reply:
I already moved the initialization of `meter` and `instrument` outside from handling flow.

Reply:
Doesn't look like it would hurt to move it out, nothing dynamic about it.  But I'd do that in a separate PR to keep this one focused.